src/main/kotlin/dsm/pick2024/domain/attendance/service/QueryClassAllAttendanceService.kt

```
@@ -3,33 +3,31 @@ package dsm.pick2024.domain.attendance.service
     @Transactional(readOnly = true)
     override fun queryClassAllAttendance(grade: Int, classNum: Int) =
         queryAttendancePort.findByGradeAndClassNum(grade, classNum)
-            .map { it ->
+            .map {
                 val userId = it.userId
                 val classroomName = try {
-                    val classroom = classroomFinderUseCase.findOKClassroomOrThrow(userId)
-                    classroom.classroomName ?: ""
+                    val classroom = queryClassroomPort.findOKClassroom(userId)
+                    classroom?.classroomName ?: ""
                 } catch (e: EmptyResultDataAccessException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)